src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmResponseFilesProcessSingleElementImpl.java

```diff
@@ -222,6 +229,71 @@ private void processUploadFile(String armInputUploadFilename, List<String> respo
         }
     }
 
+    private void getAndLogCreateRecordFile(ExternalObjectDirectoryEntity externalObjectDirectory, String createRecordFilename) {
+        if (nonNull(createRecordFilename)) {
+            Path jsonPath = null;
+            try {
+                CreateRecordFilenameProcessor createRecordFilenameProcessor = new CreateRecordFilenameProcessor(createRecordFilename);
+                BinaryData createRecordFileBinary = armDataManagementApi.getBlobData(createRecordFilenameProcessor.getCreateRecordFilenameAndPath());
+                if (nonNull(createRecordFileBinary)) {
+                    boolean appendUuidToWorkspace = true;
+                    jsonPath = fileOperationService.saveBinaryDataToSpecifiedWorkspace(
+                        createRecordFileBinary,
+                        createRecordFilenameProcessor.getCreateRecordFilename(),
+                        armDataManagementConfiguration.getTempBlobWorkspace(),
+                        appendUuidToWorkspace
+                    );
+
+                    if (nonNull(jsonPath) && jsonPath.toFile().exists()) {
+                        logResponseFileContents(jsonPath);
+                    }
+                }
+            } catch (Exception e) {
```

Comment:
It looks like only `IOException` is thrown in the try, so we can we catch that rather than catching the top-level `Exception` type?

Reply:
We don't really care what exceptions are thrown as it stops the whole process. All we are doing is logging the contents of the response file so that if the file gets deleted we have some traceability of what was in it. It helps with identifying any issues more than anything else.